src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/Util.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.SQLite
+{
+    internal static class Util
+    {
+        internal static byte[] ToUtf8(string text)
```

Comment:
This should probably be called "ToUtf8WithNullTerminator" or something. Seeing initial use my thought was "why do we need a helper?"